miso-web/src/main/webapp/scripts/bulk_sample.js

```diff
@@ -1154,7 +1154,7 @@ BulkTarget.sample = (function ($) {
       columns.push(
         {
           title: "Target Cell Recovery",
-          type: "decimal",
+          type: "int",
```

Comment:
`precision` and `scale` attributes should be removed since they are only for decimals. Can set `min` to 0 as well. Same in `form_sample.js`